styles/_form-builder.scss

```diff
@@ -1,101 +1,38 @@
 .form-builder {
   /*--------------------------------------------*
-  * Element dropdown
```

Comment:
Removing this for cleanup should have been done in https://github.com/cds-snc/platform-forms-client/pull/1843